addons/stock_account/models/stock_move.py

```diff
@@ -179,11 +179,12 @@ def _prepare_common_svl_vals(self):
         :rtype: dict
         """
         self.ensure_one()
+        description = '%s - %s' % (self.reference, self.product_id.name) if self.reference else self.product_id.name
```

Comment:
```suggestion
        if self.scrapped:
            description =  self.scrap_id._get_svl_description()
        else:
            description = '%s - %s' % (self.reference, self.product_id.name) if self.reference else self.product_id.name
```
Very picky but if we can avoid unnecessary database searches (on product name for instance) we tend to take advantage.